.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: Rust CI
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+    # This job tests on Ubuntu and macOS.
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, macos-13]
+        rust:
+          - stable
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: Swatinem/rust-cache@v2
+
+    - name: Install Rust Toolchain (stable)
+      run: |
+          rustup toolchain install stable --profile minimal
+          rustup default stable
+    
+    - name: Install protobuf compiler
+      run: |
+        if [ "${{ matrix.os }}" == "macos-13" ]; then
+          brew install protobuf
+        else
+          sudo apt-get install -y protobuf-compiler
+        fi
+
+    - name: Build civkit-cli
+      run: cargo build --bin civkit-cli --verbose
+
+    - name: Build civkitd
+      run: cargo build --bin civkitd --verbose
+
+    - name: Build civkit-sample
+      run: cargo build --bin civkit-sample --verbose
+
+  build_debian:
+    # This job tests on Debian.
+    runs-on: ubuntu-22.04
+
+    container:
+      image: debian:latest
```

Comment:
if we can pin explicitly the debian version like bullseye otherwise we’ll have serve the GA runners definition of “latest”. most of debian platforms are probably on bullseye right now

Reply:
ive added that as a comment on the code.